### ISSUE TITLE
Cask sudo prompt uses GPGTools logo

### ISSUE THREAD

**Issue by mologie** [milanvarady/Applite#43]
Hi, I received the following prompt when attempting to install Microsoft Teams through Applite:

<img width="657" alt="Screenshot 2023-10-27 at 10 52 07" src="https://github.com/milanvarady/Applite/assets/597682/4b97a388-b504-4e16-96a7-d144b32b0b12">

The lock logo is from the https://gpgtools.org/ project, and I was /super/ confused by this because I first thought it prompted for my GPG keychain passphrase, which was of course unexpected.

If anything this should pop up an Apple system authorization dialog, and if a custom one is not avoidable it should not use icons from unrelated projects, copyright issues aside.

**Comment by milanvarady** [comment]
Hi, you were prompted for your admin password because Microsoft Teams has a .pkg installer which requires it. We need to pass the password into the running brew shell instance through the 'SUDO_ASKPASS' parameter, and given that Applite is a GUI focused app we needed a graphical solution. The default apple password prompt unfortunately cannot be used through the command line, so we ended up using the ['pinentry-mac'](https://formulae.brew.sh/formula/pinentry-mac) package, which is part of GPGTools. It is the most secure way we found to pass the password into the shell. See #5 for more info.
